spec/factories.rb

```diff
@@ -0,0 +1,21 @@
+FactoryBot.define do
+
+  factory :developer, class: Developer do
+    email { Faker::Internet.email }
+  end
+
+  factory :programming_language, class: ProgrammingLanguage do
+    name 'Ruby'
+  end
+
+  factory :developer_programming_language, class: DeveloperProgrammingLanguage do
+  end
+
+  factory :language, class: Language do
+    code 'fr'
+  end
+
+  factory :developer_language, class: DeveloperLanguage do
+  end
+
+end
```

Comment:
Please check the code yourself on github before review if you haven't.
Then you can notice the warning by github.